src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.scss

```diff
@@ -87,4 +88,12 @@
             width: 100%;
         }
     }
+
+    .draft-js-editor label div {
```

Comment:
Unsure if this CSS is necessary

Reply:
I would assume that a component in DraftJS has its styles being overridden. We'll leave for now